clusters/app.ci/app.ci/prow/staging/gcsweb.yaml

```diff
@@ -24,7 +24,7 @@ items:
           name: gcsweb
           args:
           - "-b"
-          - "origin-ci-test"
```

Comment:
Use https://github.com/kubernetes/test-infra/pull/31456 and deploy with alias to `origin-ci-test` .